xml/depl_nodes.xml

```diff
@@ -4223,9 +4223,10 @@ getent group postgres | cut -d: -f3</screen>
 
   <variablelist>
     <varlistentry>
-      <term></term>
+      <term>Use Ceph deployed by Crowbar</term>
       <listitem>
 	<para>
+	  Set to <systemitem>true</systemitem> to use Ceph deployed with Crowbar.
```

Comment:
Can native English speakers suggest the correct preposition (by/with).
And make it consistent? @aspiers maybe?